modules/syn_scan/dns_grabber.go

```diff
@@ -13,7 +13,7 @@ func grabChaos(addr string, q string) string {
 	c := new(dns.Client)
 	m := new(dns.Msg)
 	m.Question = make([]dns.Question, 1)
-	m.Question[0] = dns.Question{q, dns.TypeTXT, dns.ClassCHAOS}
+	m.Question[0] = dns.Question{Name: q, Qtype: dns.TypeTXT, Qclass: dns.ClassCHAOS}
```

Comment:
Note this is unrelated to the lock copy fixes, but was an issue pointed out by `go vet ./...` so I figured I'd include it anyways.